packages/integrations/tailwind/README.md

```diff
@@ -47,22 +57,35 @@ export default {
   integrations: [tailwind()],
 }
 ```
+  
+Then, restart the dev server.
+</details>
 
-## Getting started
+## Usage
```

Comment:
Visually, this hierarchy feels a little off.

The big heading is Usage, but all the subheadings are configuration-related. And, when I see "usage" I would probably expect to see code samples, Tailwind actually being used in an Astro component. What about a screenshot here, for the people who are skimming headers and didn't watch your video? "Oh, here's where I can jump to to see how to USE Tailwind in Astro..." and then I feel like this doesn't give them that.  :)

Does this also point to maybe Configuration deserves its own heading? 

![Screenshot 2022-05-24 10 50 49](https://user-images.githubusercontent.com/5098874/170052020-23e2b156-d8d9-4033-8557-ca37f746efa9.png)

Reply:
Configuration is part of usage. In order to use something the way you want it, you must configure it

Reply:
For other integrations I absolutely agree - this is the spot for screenshots! But for this one, I feel like I got there already in the intro, and the most important point I want to make is GO TO TAILWIND DOCS 😆